egs2/accented_french_openslr56/asr1/local/normalize_test.py

```diff
@@ -0,0 +1,14 @@
+f = open('downloads/African_Accented_French/transcripts/test/ca16/prompts.txt')
```

Comment:
it's better to add a comment on how it is normalized with an example of before and after this processing

Reply:
ok